ipatests/test_integration/test_renewal_master.py

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (C) 2016  FreeIPA Contributors see COPYING for license
+#
+
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.test_integration import tasks
+
+
+class TestRenewalMaster(IntegrationTest):
+
+    topology = 'star'
+    num_replicas = 1
+
+    @classmethod
+    def uninstall(cls, mh):
+        super(TestRenewalMaster, cls).uninstall(mh)
+
+    def test_replica_not_marked_as_renewal_master(self):
+        """
+        https://fedorahosted.org/freeipa/ticket/5902
+        """
+        master = self.master
+        suffix = ",".join(["dc=%s" % i for i in master.domain.name.split('.')])
+        result = master.run_command([
+            "ldapsearch", "-Y", "GSSAPI", "-b",
+            "cn=masters,cn=ipa,cn=etc,%s" % suffix,
+            "(ipaConfigString=caRenewalMaster)"
+        ])
+        assert("numResponses: 2" in result.stdout_text), (
+            "Unexpected number of responces in"
+            " the command stdout: %s" % result.stdout_text)
+        assert("numEntries: 1" in result.stdout_text), (
+            "Unexpected number of renewal master entries in"
+            " the command stdout: %s" % result.stdout_text)
+        # Test that after master uninstallation, replica overtakes the cert
+        # renewal master role
+        tasks.uninstall_master(master)
+        result1 = self.replicas[0].run_command([
+            "ldapsearch", "-Y", "GSSAPI", "-b",
+            "cn=masters,cn=ipa,cn=etc,%s" % suffix,
+            "(ipaConfigString=caRenewalMaster)"
```

Comment:
The same here, you just assert that the value now equals to replica instead of master.